eduNEXT/edunext-platform
# feat: update to limonero.latest

## Description

Add all the commits needed to have an updated version based on open-release/lilac.master

## Changes

**Commits not applied**

1. [Update edx-proctoring](https://github.com/edx/edx-platform/commit/d61dcac29d1651956623c150be53a8bbe69e9346): they went from 3.8.5 -> 3.25.0 and that jump is very big. It ends up breaking things. Slack discussion: https://openedx.slack.com/archives/C01AGTSB1LL/p1638453201090600
2. [Add new GA workflow](https://github.com/edx/edx-platform/commit/5d786503b1262f1092245fb1c69daff918b5957f): we usually remove unused workflows as part of FFI-8.
3. [Site conf language changes](https://github.com/edx/edx-platform/commit/16e69e068e96d63228a0bcb12929a23c115f52f9): it does not allow us to change the language using the LC-V2 changes.
4. [Hide enterprise login](https://github.com/edx/edx-platform/commit/335ab58a345f2d8066b5a9793ff479fa521c4bf8): we don't use enterprise.
5. [Django update to 2.2.25](https://github.com/edx/edx-platform/commit/01027237d7b07084eb86fd4c2978b20a27d3d8ad): was breaking webpack.

These last two need more research before applying.

**Commits added by us**
1. [Add webhook-xblock to CELERY_IMPORTS](https://github.com/eduNEXT/edunext-platform/pull/539/commits/14c8695df7ddd3d352d0bfee386303ad44babc28)

## More information
- [Jira](https://edunext.atlassian.net/browse/PS2021-1384?atlOrigin=eyJpIjoiMjdjY2QxODQ2MjQ1NGI1ZmEyMGYyYWMyNTJlYjFiNjIiLCJwIjoiaiJ9)
- [Commits selection docs](https://docs.google.com/spreadsheets/d/1YsGXX6WW-m5eh78NGMVw3Z730QUJlksOvLur53FfJOU/edit#gid=260555876)

## How to test
This in stage, the important thing here is to see that nothing is broken other than checking the functionality of each backport.

Suggestions of things to try:
- Login
- Register
- Enroll / Unenroll
- Task (for example: in a course go to the Instructor Dashboard, student admin and change the grade of a problem)
- Send emails
